domains/ary.json

```
@@ -0,0 +1,13 @@
+{
+    "description": "My Portfolio Website",
+    "repo": "https://github.com/aryan2bamane/Portfolio",
+    "owner": {
+    "username": "aryan2bamane",
+    "email": "aryanbamane2@gmail.com"
+    },
+
+    "record": {
+    "CNAME": "https://aryan2bamane.github.io/Portfolio/"
```

CNAME can't contain protocol or `/`
```suggestion
    "CNAME": "aryan2bamane.github.io"
```